backend/src/main/java/ddangkong/service/balance/room/RoomService.java

```diff
@@ -70,4 +71,12 @@ private RoomContent findCurrentRoomContent(Room room) {
         return roomContentRepository.findByRoomAndRound(room, room.getCurrentRound())
                 .orElseThrow(() -> new BadRequestException("해당 방의 현재 진행중인 질문이 존재하지 않습니다."));
     }
+
+    public RoundFinishedResponse getMyRoundFinished(Long roomId, int myRound) {
+        Room room = roomRepository.getById(roomId);
+        if (room.isAllRoundFinished()) {
+            return RoundFinishedResponse.allRoundFinished();
+        }
+        return RoundFinishedResponse.myRoundFinished(room, myRound);
+    }
```

Comment:
제안) `room.isAllRoundFinished()`일 때 `true`를 반환하는 로직까지 `room.isMyRoundFinished(myRound)`에 넣는 건 어떨까요?

```suggestion
    public RoundFinishedResponse getMyRoundFinished(Long roomId, int myRound) {
        Room room = roomRepository.getById(roomId);
        return RoundFinishedResponse.myRoundFinished(room, myRound);
    }
```

Reply:
프린, 커찬, 썬데이 페어 프로그래밍에서 나눈 api 스펙 반영
-> 해당 로직에 변경이 있으므로 로직 변경 후 재리뷰하는 것으로 결정